actionview/lib/action_view/helpers/tags/hidden_field.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 
+require "action_view/helpers/tags/validator"
+
 module ActionView
   module Helpers
     module Tags # :nodoc:
       class HiddenField < TextField # :nodoc:
+        def initialize(*)
+          super
+
+          @validator = NullValidator.new
```

Comment:
`<input type="hidden">` elements can't receive focus, and are largely hidden from the [<abbr title="Accessibility Tree">AT</abbr>](https://developers.google.com/web/fundamentals/accessibility/semantics-builtin/the-accessibility-tree).